### PR TITLE
fixing IDENTITY-3551

### PR DESCRIPTION
**Pull request by dracusds123**

### CODE REVIEW

File: components/oauth/org.wso2.carbon.identity.oauth.endpoint/src/main/java/org/wso2/carbon/identity/oauth/endpoint/token/OAuth2TokenEndpoint.java [hunk 1]
```diff
@@ -199,7 +225,7 @@ private Response handleBasicAuthFailure() throws OAuthSystemException {
 
     private void logAccessTokenRequest(HttpServletRequest request) {
 
-        if (log.isDebugEnabled()){
+        if (log.isDebugEnabled()) {
```
Comment (dracusds123) [inline components/oauth/org.wso2.carbon.identity.oauth.endpoint/src/main/java/org/wso2/carbon/identity/oauth/endpoint/token/OAuth2TokenEndpoint.java]: unformatted code. Better to merge the space along with this.

File: components/oauth/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/OAuth2Service.java [hunk 2]
```diff
@@ -392,7 +397,7 @@ public OAuthRevocationResponseDTO revokeTokenByOAuthClient(OAuthRevocationReques
                 Map<String, String> extClaimsMap =
                         userStore.getUserClaimValues(username, claims,
                                 profileName);
-                for (Map.Entry<String, String> entry : extClaimsMap.entrySet()){
+                for (Map.Entry<String, String> entry : extClaimsMap.entrySet()) {
```
Comment (dracusds123) [inline components/oauth/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/OAuth2Service.java]: -do-

**Comment by johannnallathamby** [comment]
Please check for the updated fix.

**Comment by johannnallathamby** [comment]
Fixed with #1036

**Pull request closed without merging**